### DISCUSSION TITLE
Give The Wicked Altar More To Do (Expert)

### DISCUSSION THREAD

**Discussion by Flounder-M** [NillerMedDild/Enigmatica6#5117] [Ideas]
I think the Wicked Altar could be made more attractive. Throughout my playthrough I've only managed to notice 2 primary uses for the Altar: Soul Gems for Mastery Shards, and a few Items related to the Mycelial Generator. Those two purposes aside, it has the BM item management tools which seem mute as you'll have XNet by then, and several 1-off trinkets. All in all the only recipe that is worthwhile to automate in the average playthrough (and in some playthrough cases I'm sure the only reason it was made) is soul gems.

My suggestion is to add some more recipes to the Altar that allow it to become desirable. One good example, that stays on-theme with Eidolon, could be a faster/more efficient Calx of End recipe, providing an attractive alternative to the slow cooking pot. It could also offer an alternative to several BM Alchemy Table recipes, as the Alchemy Table has quite a bloated recipe list. Other possible uses that I feel stay thematically consistent: Polished Wood, Consecrated and/or Profaned Candles, and Environmental's monster-spawn-only armors.

I could be off-base here as the Altar could have a design philosophy I'm not seeing, but for me it feels more like a checkbox to mark for soul shards as you approach the end game, while the Altar multiblock is more early-mid in its requirements. When compared to the Automated Iridescent Altar it falls short as while the AIA's main purpose seems similar, that is to provide an alternative and automated altar, the outputs of the AIA are commonly used in many recipes where the Wicked Altar's are not. This is where the Wicked Altar falls short.

**Comment by MuteTiefling** [comment]
Honestly, I've been looking for a home for a better Calx recipe and hadn't thought to put it here. That's a great idea. 
I'll have to give the rest of this some thought though. 

I'll admit the whole thing feels shoehorned in and that's... well because it was. We had to do something rapidly near the end of development to prevent two major bugs in Eidolon causing issues in the pack. Most of the things it crafts were never meant to be needed in large quantities. And while the routing nodes placement may seem moot to you, not everyone wants to use xNet for everything, and this was a way to make them available earlier than they otherwise would have been with default recipes.

**Reply by Flounder-M** [reply]
Thank you for the insight on the thought behind the development there. The note on BM item management is fair.

I'll also add that I've had a blast playing through and truly appreciate the effort you, Niller, and other collaborating have given the community. Keep up the amazing work!
